mne/utils/misc.py

```diff
@@ -406,3 +406,32 @@ def _resource_path(submodule, filename):
     except ImportError:
         from pkg_resources import resource_filename
         return resource_filename(submodule, filename)
+
+
+def repr_html(f):
+    """Decorate _repr_html_ methods.
+
+    If a _repr_html_ method is decorated with this decorator, the repr in a
+    notebook will show HTML or plain text depending on the config value
+    MNE_REPR_HTML (by default "True", which will render HTML).
+
+    Parameters
+    ----------
+    f : function
+        The function to decorate.
+
+    Returns
+    -------
+    wrapper : function
+        The decorated function.
+    """
+    from ..utils import get_config
+
+    def wrapper(*args, **kwargs):
+        if get_config("MNE_REPR_HTML", "True") == "False":
```

Comment:
our config values are always strings, so there's a reasonable chance they might not have proper capitalization. Let's be permissive here by including a `.lower()` and testing against `"false"`